### PR TITLE
feat: created login and signup pages.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- added login and signup pages

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <link
+      rel="stylesheet"
+      href="https://dexterous-ui.netlify.app/stylesheets/main.css"
+    />
+    <link rel="stylesheet" href="/stylesheets/authentication.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (rahulcs754) [inline pages/login.html:11]: These tag should be top to link tag

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <div class="cart-input flex-row">
+                <input type="text" placeholder="search">
+                <a class="btn-link-noBorder" href="#">   
+                    <span class="material-icons">search</span>
+                </a>
+            </div>
+            <div class="button-box flex-row">
+              
+              <div class="box-pill">
+                <a class="btn btn-link-noBorder" href="">
```
Comment (rahulcs754) [inline pages/login.html:31]: don't leave  empty href attribute

File: pages/login.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                  <span class="material-icons">favorite_border</span>
+                </a>
+              </div>
+              <div class="box-pill">
+                <a class="btn btn-link-noBorder" href="">
```
Comment (rahulcs754) [inline pages/login.html:36]: same above

File: pages/signup.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            <div class="cart-input flex-row">
+                <input type="text" placeholder="search">
+                <a class="btn-link-noBorder" href="#">   
+                    <span class="material-icons">search</span>
+                </a>
+            </div>
+            <div class="button-box flex-row">
+              
+              <div class="box-pill">
+                <a class="btn btn-link-noBorder" href="">
```
Comment (rahulcs754) [inline pages/signup.html:31]: don't leave empty

**Review by rahulcs754 — APPROVED** [review]
looks good all code 
just few suggestion please check comment

**Review by Sameony — APPROVED** [review]
Nice code, Loved the way you used the grouping of selectors to overwrite the properties of btn classes.

**Pull request merged**